src/uv-common.c

```diff
@@ -797,3 +797,18 @@ void uv_os_free_environ(uv_env_item_t* envitems, int count) {
 
   uv__free(envitems);
 }
+
+
+void uv_free_cpu_info(uv_cpu_info_t* cpu_infos, int count) {
+#if defined(__CYGWIN__)
+  (void)cpu_infos;
+  (void)count;
```

Comment:
`uv_cpu_info()` sets them both to zero/NULL so I don't think you have to special-case for cygwin here.